docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -107,5 +107,9 @@
 
         - name: Apply Deployment Configuration File
           shell: KUBECONFIG=/etc/kubernetes/admin.conf /bin/kubectl apply -f deployment-config.yaml
+          register: apply_deployment_config
+          retries: 5
+          delay: 10
+          until: apply_deployment_config.rc == 0
```

Comment:
Presumably, you could get rid of the 10 second delay that occurs just ahead of this now, letting the first apply fail and allowing the retries to handle it, but I think this is fine and is consistent with current behaviour